components/tidb_query/src/rpn_expr/impl_string.rs

```diff
@@ -423,6 +423,36 @@ pub fn trim_1_arg(arg: &Option<Bytes>) -> Result<Option<Bytes>> {
     }))
 }
 
+#[rpn_fn]
+#[inline]
+pub fn trim_2_args(s: &Option<Bytes>, remstr: &Option<Bytes>) -> Result<Option<Bytes>> {
+    // As per MySQL doc `https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_trim`,
```

Comment:
If the user does not provide `remstr`, the function signature should be `Trim1Arg`.
Actually if `remstr` is `NULL`, the output should be `NULL`:

```
MySQL [(none)]> select trim(NULL from 'abc');
+-----------------------+
| trim(NULL from 'abc') |
+-----------------------+
| NULL                  |
+-----------------------+
1 row in set (0.000 sec)
```

Reply:
Hi @sticnarf , sorry for my bad understanding, I update the code. Please help review. Thanks.